src/backend/api/Fusion.Resources.Logic/Requests/Commands/ResourceAllocationRequest/UpdateOrgPositionInstanceHaveRequest.cs

```diff
@@ -0,0 +1,73 @@
+﻿using Fusion.ApiClients.Org;
+using MediatR;
+using Newtonsoft.Json.Linq;
+using System;
+using System.Linq;
+using System.Threading;
+using System.Threading.Tasks;
+using Fusion.Resources.Domain.Queries;
+
+namespace Fusion.Resources.Logic.Commands
+{
+    public class UpdateOrgPositionInstanceHaveRequest : IRequest
```

Comment:
We already have notifications for `InternalRequestCreated` and `InternalRequestDeleted`. Could add a handler for both these notifications and set hasRequest = true false based on type of notification instead of adding a new request issued by controllers. I.e. 

```cs
UpdateOrgPositionInstanceHaveRequestHandler : INotificationHandler<InternalRequestCreated>, INotificationHandler<InternalRequestDeleted>
{
     Handle(InternalRequestCreated...) => Update(..., hasRequest: true);
     Handle(InternalRequestDeleted...) => Update(..., hasRequest: false);
}
```

Reply:
Could utilize this yes. Agree :-)
Revoved from controller